src/components/TodoItem.js

```diff
@@ -0,0 +1,50 @@
+import React, {useRef} from 'react';
+import {AiFillEdit} from 'react-icons/ai';
+import { IoCheckmarkDoneSharp, IoClose } from 'react-icons/io5';
+import styles from '../styles/main.scss';
+
+const TodoItem = (props) => {
+  const {item, updateTodo, removeTodo, completeTodo} = props;
```

Comment:
it is better to destructure props inside an arguments of component:

const TodoItem = ({item, updateTodo, removeTodo, completeTodo}) => {}